core/citrus-api/src/test/java/org/citrusframework/spi/ClassPathResourceResolverTest.java

```diff
@@ -30,17 +42,30 @@ void loadFromFatJar() throws IOException {
                 Path.of("META-INF/citrus/test/parser/core/xpath-data-dictionary")));
         } finally {
             Thread.currentThread().setContextClassLoader(contextClassLoader);
-        }
+        }    }
+
+    static Stream<Arguments> loadFromFatJarSpringBoot() {
```

Comment:
I would place the supplier method above the test, for better readability.

Reply:
I wouldn't but I did.